src/components/dev-hub/tab.js

```diff
@@ -0,0 +1,116 @@
+import { React, useState } from 'react';
+import styled from '@emotion/styled';
+import { colorMap, size } from './theme';
+import { P } from './text';
+import CardList from './card-list';
+import useAllVideos from '../../hooks/use-all-videos';
+import usePodcasts from '../../hooks/use-podcasts';
+
+const Tab = styled('div')`
```

Comment:
We should let a component using the `Tab` define these values as needed (through using `styled` or something) like the media page, so these will likely move out of this component.

This lets us tweak the tab to fit a specific page's liking much more easily and also remove some code from the component itself.